js/npm-debug.log

```diff
@@ -0,0 +1,23 @@
+0 info it worked if it ends with ok
```

Comment:
Could you delete this file? I think you did not commit it on purpose. I guess we should put this file in `.gitignore`